### PR TITLE
add auto-map, single input to multi in sql script

### PR DESCRIPTION
**Pull request by aidanwobrien**
Add a rough 'auto-map' script which takes the layers in the layer tree and makes a user defined print layout and map title.

I also attempted to combine your single input dialog class in the one_service_with_sql script into a single dialog class with multiple text inputs that are then passed into the functions all at once

Also cleaned up some functions

### CODE REVIEW

File: load_lyrs_from_geohub_services.py
Comment (frasemcl) [inline load_lyrs_from_geohub_services.py:1]: It seems like the call to init_project() in separate module didn't quite work. Is it working for you @aidanwobrien ? "exec(Path('_FM_PATH_.../load_lyrs_from_geohub_services.py').read_text())
Loading layer...
Starting script...
Zoom warning accepted. Selecting layers of interest
No features exist: skipped layer ['11', 'Tile Drainage Area', '01']
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "<string>", line 183, in <module>
  File "<string>", line 75, in rest_request
NameError: name 'projInstance' is not defined"

**Review by frasemcl — CHANGES_REQUESTED** [review]
I remember it being harder than I expected to get one_rest_request working from a module script, let me know if it's working for you and you want me to test further. Otherwise can you tweak how init_project() works before we merge this branch? load_lyrs_from... currently fails for me on this branch.

File: load_lyrs_from_geohub_services.py
Comment (aidanwobrien) [reply load_lyrs_from_geohub_services.py:1]: hm, you're right. I thought I had the module working, but I must not have tested it all the way through.

I tried a bit to get the module working, but it's still giving me problems. LIkely something to do with QGIS / Python parameters.

For now I will remove the module and move the functions back into the scripts

**Review by aidanwobrien — COMMENTED** [review]
I removed func_service.py for now and put the functions back into the main load_lyrs script

**Review by frasemcl — CHANGES_REQUESTED** [review]
Can we keep the func_service.py module even if we're not using it in the main scripts yet? Removing it broke 'one_service_with_sql.py' since I did get the module working for that one. I was clunky looking but took some time to sort out and might help us get a modular structure working overall later on if we reference what I did there and improve on it. I propose we keep that file for now, but I'm good with the other script not using func_service.py for now. Thoughts?

**Comment by aidanwobrien** [comment]
I'm not exactly sure how to address your requested change, but I've added back in func_service.py in the newest pull request

**Comment by frasemcl** [comment]
Thanks I just didn't want to lose track of the module pattern that worked for me earlier, I'll merge now.

**Review by frasemcl — APPROVED** [review]

**Pull request merged**